packages/ui-core/src/components/side-menu/side-menu.component.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+import { MenuItem, Menu } from './side-menu.styles';
+import { Title } from '../../typography';
```

Comment:
can we use Title from `keen.io/typography`?
typography in `ui-core` should be discussed as for now it's a duplicate of `keen.io/typogpraphy`